pkg/gds/peers/v1/peers.go

```diff
@@ -0,0 +1,7 @@
+package peers
+
+import "fmt"
+
+func (p *Peer) Key() string {
```

Comment:
Thank you so much for adding the `Key` method; this absolutely helps keep the store interface clean while simplifying how we construct a key in the future.